src/IO/AsciiPCLoader/asciipcloader.hpp

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <Core/Asset/FileLoaderInterface.hpp>
+#include <IO/RaIO.hpp>
+
+using std::string;
+using std::vector;
+
+namespace Ra {
+namespace IO {
+
+class RA_IO_API AsciiPointCloudLoader : public Ra::Core::Asset::FileLoaderInterface
+{
+  public:
+    AsciiPointCloudLoader()          = default;
+    virtual ~AsciiPointCloudLoader() = default;
+
+    vector<string> getFileExtensions() const override;
+    bool handleFileExtension( const std::string& extension ) const override;
+
+    /** load vertices properties from ASCII .txt file
+     *
+     * expected content in file: header line + data records
+     *
+     * header line content : attribute names separated by blanks ( )
+     *
+     * example: Time X Y Z Roll Pitch Heading sdX sdY sdZ
+     *
+     * X, Y, Z and Time properties (case sensitive) are mandatory
```

Comment:
I am wondering if we could relax the constraint on the time field: it could be useful to load point-clouds with xyz coordinates, even if we don't have the time field.

Reply:
done. now only position properties are mandatory